src/lib/marquee.jsx

```diff
@@ -107,7 +107,7 @@ export function Marquee({
     <div
       {...props}
       ref={container}
-      className={classnames(["marquee", className])}
+      className={classnames(["marquee", marqueeWidth > 0 && "ready", className])}
```

Comment:
Let's use the BEM standards here and make the class name `marquee--ready`.  And update the styles so you don't have to increase specification.

Reply:
Thanks @marlonmarcello , done!